client/js/src/main/scala/zio/zmx/client/frontend/views/VegaChart.scala

```diff
@@ -1,22 +1,20 @@
 package zio.zmx.client.frontend.views
 
-import scala.scalajs.js
 import com.raquo.laminar.api.L._
+import org.scalajs.macrotaskexecutor.MacrotaskExecutor.Implicits._
```

Comment:
Got rid of the following warning. It probably comes from the recent releases of Scala.js:

```
[warn] zio-zmx/client/js/src/main/scala/zio/zmx/client/frontend/views/VegaChart.scala:14:39: The global execution context in Scala.js is based on JS Promises (microtasks).
[warn] Using it may prevent macrotasks (I/O, timers, UI rendering) from running reliably.
[warn]
[warn] Unfortunately, there is no way with ECMAScript only to implement a performant
[warn] macrotask execution context (and hence Scala.js core does not contain one).
[warn]
[warn] We recommend you use: https://github.com/scala-js/scala-js-macrotask-executor
[warn] Please refer to the README.md of that project for more details regarding
[warn] microtask vs. macrotask execution contexts.
[warn]
[warn] If you do not care about macrotask fairness, you can silence this warning by:
[warn] - Adding @nowarn("cat=other") (Scala >= 2.13.x only)
[warn] - Setting the -P:scalajs:nowarnGlobalExecutionContext compiler option
[warn] - Using scala.scalajs.concurrent.JSExecutionContext.queue
[warn]   (the implementation of ExecutionContext.global in Scala.js) directly.
[warn]
[warn] If you do not care about performance, you can use
[warn] scala.scalajs.concurrent.QueueExecutionContext.timeouts().
[warn] It is based on setTimeout which makes it fair but slow (due to clamping).
[warn]
[warn]     scala.concurrent.ExecutionContext.global
```